internal/bminventory/inventory.go

```diff
@@ -6613,6 +6626,7 @@ func (b *bareMetalInventory) updateHostName(ctx context.Context, host *common.Ho
 			*hostname, host.ID, host.InfraEnvID)
 		return common.NewApiError(http.StatusConflict, err)
 	}
+	b.setUsage(true, usage.RequestedHostnameUsage, &map[string]interface{}{"from_localhost": localhost}, usages)
```

Comment:
What is `from_localhost` supposed to mean?
Do we even really need extra data in this case?

If the usage is set to true then a host has requested a hostname other than `localhost`, right?